src/JustEat.StatsD/IStatsDPublisherExtensions.cs

```diff
@@ -12,14 +12,50 @@ public static class IStatsDPublisherExtensions
     {
         private const double DefaultSampleRate = 1.0;
 
+        /// <summary>
+        /// Publishes a counter for the specified bucket and value without tags.
+        /// </summary>
+        /// <param name="publisher">The <see cref="IStatsDPublisher"/> to publish with.</param>
+        /// <param name="value">The value to increment the counter by.</param>
+        /// <param name="sampleRate">The sample rate for the counter.</param>
+        /// <param name="bucket">The bucket to increment the counter for.</param>
+        public static void Increment(this IStatsDPublisher publisher, long value, double sampleRate, string bucket)
+        {
+            publisher.Increment(value, sampleRate, bucket, null);
+        }
+
+        /// <summary>
+        /// Publishes a gauge for the specified bucket and value without tags.
+        /// </summary>
+        /// <param name="publisher">The <see cref="IStatsDPublisher"/> to publish with.</param>
+        /// <param name="value">The value to publish for the gauge.</param>
+        /// <param name="bucket">The bucket to publish the gauge for.</param>
+        public static void Gauge(this IStatsDPublisher publisher, double value, string bucket)
+        {
+            publisher.Gauge(value, bucket, null);
+        }
+
+        /// <summary>
+        /// Publishes a timer for the specified bucket and value without tags.
+        /// </summary>
+        /// <param name="publisher">The <see cref="IStatsDPublisher"/> to publish with.</param>
+        /// <param name="duration">The value to publish for the timer.</param>
+        /// <param name="sampleRate">The sample rate for the timer.</param>
+        /// <param name="bucket">The bucket to publish the timer for.</param>
+        public static void Timing(this IStatsDPublisher publisher, long duration, double sampleRate, string bucket)
+        {
+            publisher.Timing(duration, sampleRate, bucket, null);
+        }
+
         /// <summary>
         /// Publishes a counter for the specified bucket with a value of one (1).
         /// </summary>
         /// <param name="publisher">The <see cref="IStatsDPublisher"/> to publish with.</param>
         /// <param name="bucket">The bucket to increment the counter for.</param>
-        public static void Increment(this IStatsDPublisher publisher, string bucket)
+        /// <param name="tags">The tags to publish with the counter.</param>
```

Comment:
Nit: on this one (and all the other new parameters) for consistency with the others should be `tag(s)`.

```suggestion
        /// <param name="tags">The tag(s) to publish with the counter.</param>
```